shopgate.php

```diff
@@ -171,6 +173,8 @@ public function install()
             'header',
             'adminOrder',
             'updateOrderStatus',
+            'updateQuantity',
```

Comment:
I found this documentation: http://doc.prestashop.com/display/PS15/Auto-updating+modules

Is this what you are talking about? How can I test if my implementation works? I think the "Update" only appears if my module is already in the market, right? So how to test it?